test_issue_fix.py

```
@@ -0,0 +1,48 @@
+
+import os
+import sys
+import logging
+
+# 로깅 설정
+logging.basicConfig(level=logging.DEBUG)
+
+# 현재 디렉토리의 src를 Python 경로에 추가
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'src'))
+
+from pyhub.llm import OpenAILLM
```

Ruff (0.11.9)

Module level import not at top of file

(E402)